tensorflow/python/kernel_tests/nn_ops/morphological_ops_test.py

```diff
@@ -180,8 +182,20 @@ def testDilation(self):
       self._testDilationSamePaddingRate(use_gpu)
       self._testDilationValidPaddingUnevenStride(use_gpu)
 
+  def testDilationBfloat16(self):
+    for use_gpu in True, False:
+      self._VerifyValues(
+          image=[[[[.1], [.2]], [[.3], [.4]]]],
+          kernel=[[[.4], [.3]], [[.1], [.0]]],
+          strides=[1, 1],
+          rates=[1, 1],
+          padding="VALID",
+          out=[[[[.5]]]],
+          use_gpu=use_gpu,
+          dtype=dtypes.bfloat16)
```

Comment:
Thanks for the RP!
ideally, we should parameterize existing tests with different dtype, instead of adding a separate test case for each dtype.

Reply:
+1

Reply:
Thanks @vufg and @cantonios for reviewing! I've parameterized the tests.